src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,77 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
+import { TodoStatus } from '../../types/TodoStatus';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  onTodoStatus: (option: TodoStatus) => void,
```

Comment:
let's check the variable name, for now it's not clear what's the main action of this function
is it called on `TodoStatus` change, or is it change `TodoStatus`?